### PR TITLE
RM18875 : STOCKMOVE get NPE when print a deleted move with the icon b…

### PR DESCRIPTION
**Pull request by tga-axelor**
…utton

### CODE REVIEW

File: axelor-stock/src/main/java/com/axelor/apps/stock/web/StockMoveController.java [hunk 1]
```diff
@@ -162,12 +162,23 @@ public Long apply(@Nullable Object input) {
                 });
         fileLink = stockMovePrintService.printStockMoves(ids);
         title = I18n.get("Stock Moves");
-      } else if (context.get("id") != null) {
-        StockMove stockMove = request.getContext().asType(StockMove.class);
-        stockMove = stockMoveRepo.find(stockMove.getId());
-        title = stockMovePrintService.getFileName(stockMove);
-        fileLink = stockMovePrintService.printStockMove(stockMove, ReportSettings.FORMAT_PDF);
-        logger.debug("Printing " + title);
+      } else if (context.get("id") != null)
```
Comment (ale-axelor) [inline axelor-stock/src/main/java/com/axelor/apps/stock/web/StockMoveController.java]: Please apply google code format using spotless task.

File: axelor-stock/src/main/resources/i18n/messages_fr.csv [hunk 2]
```diff
@@ -682,6 +682,7 @@
 "There's no destination stock location selected for the stock's movement %s","Aucun emplacement destination selectionné pour le mouvement de stock %s",,
 "There's no product in stock location.","Il n'y a aucun produit contenu dans l'emplacement de stock.",,
 "There's no source stock location selected for the stock's movement %s","Aucun emplacement source selectionné pour le mouvement de stock %s",,
+"This move has been edited, please refresh.","Le mouvement de stock a été modifier, veuillez rafraichir la page.",,
```
Comment (ale-axelor) [inline axelor-stock/src/main/resources/i18n/messages_fr.csv]: a été modifier => a été modifié

File: CHANGELOG.md [hunk 3]
```diff
@@ -22,6 +22,7 @@
 - SALE ORDER LINE: Fix NPE in groovy script.
 - PACKING LIST : address not readable on printing
 - HUMAN RESOURCE DEMO DATA : holiday planning is now correctly imported.
+- STOCKMOVE : catch NPE when print with icon a deleted move.
```
Comment (ale-axelor) [inline CHANGELOG.md]: => catch NPE when printing a deleted move.

Comment (tga-axelor) [reply CHANGELOG.md]: the error was not triggered by the toolbar

**Pull request closed without merging**